src/Services/Catalog/CatalogService.WebApi/Controllers/AuthorController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using CatalogService.Application.Dto;
+using CatalogService.Application.UseCases;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CatalogService.WebApi.Controllers
+{
+    [ApiController]
+    [Route("author")]
+    public class AuthorController(
+        IMediator mediator) : ControllerBase
+    {
+        private readonly IMediator _mediator = mediator;
+
+        [HttpPost]
+        public async Task<IActionResult> Create([FromBody] CreateAuthorCommand request, CancellationToken token = default)
```

Comment:
Set in all your Create, Update, Delete endpoints role based access

there are roles "admin", "seller", "customer", use a class with constant props, it will be easy to change if required